smokeTest/gameOfThrones.cpp

```diff
@@ -0,0 +1,56 @@
+/* The intention of this code is to solve the Game Of Thrones challenge from hackerrank
+ * The challenge can be seen here -> https://www.hackerrank.com/challenges/game-of-thrones/problem?h_r=internal-search
+ * Author: Lucas Lovato
+ * Date: 08/04/2020
+ * Update:
+*/
+
+#include <bits/stdc++.h>
+
+#define SIZE 255
+using namespace std;
+
+int LETTERS_ARRAY[SIZE];
```

Comment:
- global variables are evil: https://www.learncpp.com/cpp-tutorial/why-global-variables-are-evil/
- you could declare the variable in main and then pass a pointer as necessary